Sources/Sentry/SentryMetricProfiler.mm

```diff
@@ -48,7 +48,7 @@ @implementation SentryMetricReading
     [absoluteTimestampValues enumerateObjectsUsingBlock:^(
         SentryMetricReading *_Nonnull reading, NSUInteger idx, BOOL *_Nonnull stop) {
         // if the metric reading wasn't recorded until the transaction ended, don't include it
-        if (orderedChronologically(transaction.endSystemTime, reading.absoluteTimestamp)) {
```

Comment:
This would throw out a measurement even if it happened at the time the transaction ended. I figure we might want to keep that data instead. Flipping parameters and negating changes testing a <= b instead to a > b, which is really the case we want to skip (measurement came strictly after transaction).